### PR TITLE
Added CLI related merged PRs to the documentation

### PR DESCRIPTION
**Pull request by didierjoomun**
DEPRECATED message updated with warning sign and DEPRECATED text adde to the page title.

### CODE REVIEW

**Comment by igorpecovnik** [comment]
Lets merge this once there are a few of new instructions ready. So people see its not only this "deprecated" sign ;)

**Comment by didierjoomun** [comment]
Hi @igorpecovnik 
Added the following PRs to the documentation (/Developer-Guide_Build-Options/):

- https://github.com/armbian/build/pull/6482
- https://github.com/armbian/build/pull/6376 and https://github.com/armbian/build/pull/6392
- https://github.com/armbian/build/pull/6354
- https://github.com/armbian/build/pull/6353

**Comment by igorpecovnik** [comment]
Thank you, this looks like a good start!

@EvilOlaf @rpardini @lanefu @ColorfulRhino ... 

Any suggestions?
Can we now start to block new features merge when they come without such entry?

**Comment by EvilOlaf** [comment]
I suggest to change wording from _options_ to _commands_ since this is what those are.

**Comment by lanefu** [comment]
> I suggest to change wording from _options_ to _commands_ since this is what those are.

Maybe leave the heading there to say build options have been replaced with build commands and have build command section immediately below.

File: docs/Developer-Guide_Build-Options.md [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+Usage:
+```bash
+./compile.sh rewrite-uboot-patches BOARD=xxxx BRANCH=edge 
+```
+
+### targets
+
+Generates output/git_sources.json file that will contain url, branch and commit hash combo.
+
+The easiest way to generate file for all devices is to run `./compile.sh targets`. Then at the time of release we will copy the output/info/git_sources.json file to config/sources/git_sources.json. Once the file is copied, the hash information from the file will be used to fetch resources for git repositories where branches are specified instead of tags or commits.
```
Comment (rpardini) [inline docs/Developer-Guide_Build-Options.md:54]: This... is not what `targets` does?

Comment (igorpecovnik) [reply docs/Developer-Guide_Build-Options.md:54]: Description https://github.com/armbian/build/pull/6272

**Review by igorpecovnik — APPROVED** [review]
I think we can merge this part.

**Pull request merged**